### PR TITLE
I/6107: Add RTL support for table (cell) properties

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Added RTL support for table and table cell properties. Closes ckeditor/ckeditor5#6107.

--

Related: https://github.com/ckeditor/ckeditor5-theme-lark/pull/269

### CODE REVIEW

**Comment by oleq** [comment]
[I also found this bug when reviewing the UI.](https://github.com/ckeditor/ckeditor5/issues/6371). Do you think @Reinmar we can handle this in the same it?

**Comment by Reinmar** [comment]
> [I also found this bug when reviewing the UI](https://github.com/ckeditor/ckeditor5/issues/6371). Do you think @Reinmar we can handle this in the same it?

We don't have capacity now. But let's keep this in mind.

File: src/tablecellproperties/ui/tablecellpropertiesview.js [hunk 1]
```diff
@@ -783,12 +784,29 @@ export default class TableCellPropertiesView extends View {
...
 			right: t( 'Align cell text to the right' ),
 			justify: t( 'Justify cell text' )
 		};
+
+		// Set of labels which can be reversed to match proper {@link #uiLanguage editor UI language}.
+		const ltr = [ 'left', 'center', 'right' ];
+		const labelsDirection = this.locale.uiLanguageDirection === 'rtl' ? ltr.reverse() : ltr;
+
+		// Creates object with a proper order of labeles.
+		const orderedLabels = labelsDirection.reduce( ( acc, curr ) => {
```
[the remark this answers is not in the document]
Comment (panr) [reply src/tablecellproperties/ui/tablecellpropertiesview.js]: crushed

Oh, boy... You're right...

File: src/tablecellproperties/ui/tablecellpropertiesview.js [hunk 2]
```diff
@@ -783,12 +784,29 @@ export default class TableCellPropertiesView extends View {
...
 
-		return {
+		const labels = {
 			left: t( 'Align cell text to the left' ),
 			center: t( 'Align cell text to the center' ),
 			right: t( 'Align cell text to the right' ),
 			justify: t( 'Justify cell text' )
 		};
+
+		// Set of labels which can be reversed to match proper {@link #uiLanguage editor UI language}.
```
[the remark this answers is not in the document]
Comment (panr) [reply src/tablecellproperties/ui/tablecellpropertiesview.js]: Removed, it's no longer necessary here.

File: src/tablecellproperties/ui/tablecellpropertiesview.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (panr) [reply src/tablecellproperties/ui/tablecellpropertiesview.js]: Kissed 😗

**Comment by panr** [comment]
@oleq all fixed!

**Pull request merged**